orttraining/orttraining/test/python/orttraining_run_frontend_batch_size_test.py

```diff
@@ -0,0 +1,56 @@
+import sys
+import collections
+import subprocess
+
+Config = collections.namedtuple(
+    "Config",
+    [
+        "enable_mixed_precision",
+        "sequence_length",
+        "max_batch_size",
+        "max_predictions_per_seq",
+        "gelu_recompute",
+        "attn_dropout_recompute",
+        "transformer_layer_recompute"])
+
+configs = [
+    Config(True, 128, 46, 20, False, False, False),
+    Config(True, 512, 8, 80, False, False, False),
+    Config(False, 128, 26, 20, False, False, False),
+    Config(False, 512, 4, 80, False, False, False),
+    Config(True, 128, 50, 20, True, False, False),
+    Config(True, 128, 50, 20, False, True, False),
+    Config(True, 128, 76, 20, False, False, True),
+    Config(True, 512, 8, 80, True, False, False),
+    Config(True, 512, 9, 80, False, True, False),
+    Config(True, 512, 15, 80, False, False, True),
+]
+
+def run_with_config(config):
+    print("##### testing name - {}-{} #####".format("fp16" if config.enable_mixed_precision else "fp32",
+                                                        config.sequence_length))
+    print("gelu_recompute: ", config.gelu_recompute)
+    print("attn_dropout_recompute: ", config.attn_dropout_recompute)
+    print("transformer_layer_recompute: ", config.transformer_layer_recompute)
```

Comment:
Are these intended or debugging?

Reply:
It is helpful to see what is being tested. Yes it is intended.